### DISCUSSION TITLE
Running Postal on Centos-arm

### DISCUSSION THREAD

**Discussion by rgmhtt** [postalserver/postal#1879] [Help with using Postal] [answered]
I have read the post on running Postal on a Raspberry.  I want to run it on a Odroid HC1 with Centos-arm 8.5 (RedSleeve build).  The HC1 only has 2GB memory, but is an octo core.

I am currently running a self-built mailserver with Postfix, Dovecot, MariaDB (for user accts only, messages are in separate files) on a CubieTruck with 2GB/duo core.  free reports 1.8GB used and .9GB swap.  So it 'works' for the number of users and email I receive.

Now on to questions:

Is there any specific OS related module compile requirements?
If the messages are in a single SQL database, how is backup managed?

I am sure there was another question, but it is gone for now.
Next week, I may  be able to spin up the Odroid and make a test to for a test.domain.

**Comment by willpower232** [comment, marked as answer]
Postal V2 uses docker so as long as you can run that on your device of choice, it will at least try to run. I think some others have had issues running Postal on ARM but I can't remember what the exact problem was so I won't guarantee that it will work :sweat_smile: 

Its worth bearing in mind that Postal is not a traditional email server with mailboxes and accounts, it is mostly designed for sending messages and by default deletes the message records it does have after a period of time.

Backing up Postal requires you to take backups of the whole database server as there are multiple databases involved. Postal does not have any backups of its own.

**Reply by rgmhtt** [reply]
OK.  So this is not the package I am looking for to replace me hand-crafted mail server.  I just may end up doing that again.  It is actually the anti-virus that is the hard piece.  Others are rather straightforward.

take care.
